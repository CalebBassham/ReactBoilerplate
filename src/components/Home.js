import React, { Component } from 'react';

export class Home extends Component {
  render() {
    return (
      <h1>&lt;ReactBoilerplate /&gt;</h1>
    )
  }
}

export default Home;