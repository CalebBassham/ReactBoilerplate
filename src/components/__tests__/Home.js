import React from 'react';
import { shallow } from 'enzyme';
import Home from '../Home';

describe('Home Component', () => {
  it('should show hello message', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot()
  })
})