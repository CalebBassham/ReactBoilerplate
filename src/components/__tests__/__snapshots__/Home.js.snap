// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component should show hello message 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Home />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": "<ReactBoilerplate />",
    },
    "ref": null,
    "rendered": "<ReactBoilerplate />",
    "type": "h1",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": "<ReactBoilerplate />",
      },
      "ref": null,
      "rendered": "<ReactBoilerplate />",
      "type": "h1",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
